dashboard/client/src/components/StatusChip.tsx

```diff
@@ -20,15 +20,15 @@ const grey = "#5F6469";
 const colorMap = {
   node: {
     ALIVE: green,
-    DEAD: red,
+    DEAD: grey,
```

Comment:
Discussed with @scottsun94; changed the color of the dead entity to grey.
I made the change in this PR to reduce context switching. I will submit an independent PR next time.